src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -50,13 +51,12 @@ internal IArgumentOperation CreateArgumentOperation(ArgumentKind kind, IParamete
                 syntax,
                 isImplicit);
         }
-#nullable disable
 
-        internal IVariableInitializerOperation CreateVariableDeclaratorInitializer(BoundLocalDeclaration boundLocalDeclaration, SyntaxNode syntax)
+        internal IVariableInitializerOperation? CreateVariableDeclaratorInitializer(BoundLocalDeclaration boundLocalDeclaration, SyntaxNode syntax)
```

Comment:
Should we add a `NotNullIfNotNull("boundLocalDeclaration")`?

Reply:
Certainly could. It wasn't necessary for usage so I didn't add it.

Reply:
Actually, no, it's if the there is an initializer that this will return non-null. Can't use an attribute to declared that relationship.